features/scope-pseudo.yml

```diff
@@ -0,0 +1,7 @@
+name: :scope pseudo-class
+description: The `:scope` CSS pseudo-class represents the scoping root, for instance the element that `querySelector()` is called on.
```

Comment:
```suggestion
description: The `:scope` CSS pseudo-class matches the scoping root, for instance the element that `querySelector()` is called on.
```

Reply:
This description should also mention that the pseudo-class is useful when used inside `@scope` blocks.

MDN:
> When used inside a [@scope](https://developer.mozilla.org/en-US/docs/Web/CSS/@scope) block, :scope matches the block's defined scope root.

https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

Reply:
Good catch. Added.